sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/GatewayAddressCache.java

```diff
@@ -884,7 +884,7 @@ private void validateReplicaAddresses(String collectionRid, AddressInformation[]
 
         if (addressesNeedToValidation.size() > 0) {
             this.openConnectionsHandler
-                    .openConnections(collectionRid, this.serviceEndpoint, addressesNeedToValidation)
+                    .openConnections(collectionRid, this.serviceEndpoint, addressesNeedToValidation, Configs.getMinConnectionPoolSizePerEndpoint())
                     .subscribeOn(CosmosSchedulers.OPEN_CONNECTIONS_BOUNDED_ELASTIC)
```

Comment:
What should be a good value for opening connections for addresses that need validation? Defaulting to system config setting for now.